tasks/python-reference-activity/README.md

```diff
@@ -0,0 +1,32 @@
+# @cumulus/python-reference-activity
+
+This 'task' is a reference activity implementation that is included with Cumulus to allow integration testing of the `Cumulus message adapter` and [`cumulus-process-py`](https://github.com/nasa/cumulus-process-py) with a deployed activity.
```

Comment:
```suggestion
This 'task' is a reference activity implementation that is included with Cumulus to allow integration testing of the `cumulus-message-adapter` and [`cumulus-process-py`](https://github.com/nasa/cumulus-process-py) with a deployed activity.
```

Reply:
`cumulus-message-adapter` should be linked

Reply:
5d7678a8f